src/views/Settings/TaxesForm.tsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from 'react';
+import { Redirect } from 'react-router-dom';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { Button, useToast } from '@chakra-ui/react';
+
+import { ROUTE_SETTINGS } from 'routing';
+
+import { Tax } from 'model/documents/tax';
+
+import { requiredErrorMessage } from 'others/form-default-errors';
+
+import FormTemplate from 'components/templates/FormTemplate';
+import FormGroupInput from 'components/molecules/FormGroupInput';
+import FormGroupSelect from 'components/molecules/FormGroupSelect';
+import FormGroupNumber from 'components/molecules/FormGroupNumber';
+import FormGroupCheckbox from 'components/molecules/FormGroupCheckbox';
+import { saveAll } from 'api/firebase/firestore/firestore-actions';
+
+const TaxesForm = () => {
+  const [isSubmitted, setIsSubmitted] = useState<boolean>(false);
+  const toast = useToast();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting },
+  } = useForm<Tax>();
+
+  const onSubmit: SubmitHandler<Tax> = (data: Tax) => {
+    saveAll('taxes', data).then((ref) => {
+      setIsSubmitted(true);
+      toast({
+        title: 'Podatek dodany 🙌',
+        status: 'success',
+        duration: 5000,
+        isClosable: true,
+        position: 'bottom-right',
+      });
+    });
+  };
+
+  const TAX_OPTIONS = ['Wartość dodana (VAT)', 'Z obrotu'];
```

Comment:
As I mentioned in another comment - in my opinion that should be `enum`.

Reply:
Used `Object.values` on newly created enum `TaxType` instead of hardcoding this array.